css-box-model/styles.css

```diff
@@ -0,0 +1,12 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+}
+
+button {
+  margin: 0 12px;
+  padding: 6px 16px;
+  border-radius: 6px;
+}
```

Comment:
You didn't need to create a new rule here, you can add your styling to the above rule.